views/layouts/main.hbs

```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <link rel="stylesheet" type="text/css" href="./style/style.css">
```

Comment:
might be good to add a meta viewport tag - see here https://www.w3schools.com/css/css_rwd_viewport.asp